sihl_contrib/session/test/run_mariadb.ml

```diff
@@ -0,0 +1,18 @@
+let config =
+  Sihl.Core.Config.Setting.create ~development:[]
+    ~test:
+      [
+        ("BASE_URL", "http://localhost:3000");
+        ("DATABASE_URL", "postgres://admin:password@127.0.0.1:5432/dev");
```

Comment:
I suppose the URL should be `mariadb...`

Reply:
Yep, and the port should be 3306 :) Fixed.